<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PasswordEntryDialog</class>
 <widget class="QDialog" name="PasswordEntryDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>428</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dodaj</string>
  </property>
  <widget class="QPushButton" name="add_button">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>350</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Dodaj</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="name_line_edit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>301</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="username_line_edit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>301</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="password_line_edit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>200</y>
     <width>301</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>180</y>
     <width>57</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Hasło</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>301</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Nazwa użytkownika lub email</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>57</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Nazwa</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
